src/main/java/com/labsynch/labseer/domain/Lot.java

```diff
@@ -614,10 +614,11 @@ public static TypedQuery<LotsByProjectDTO> findLotsByProjectsList(List<String> p
 
         EntityManager em = Lot.entityManager();
         String query = "SELECT new com.labsynch.labseer.dto.LotsByProjectDTO( "
-                + "lt.id as id, lt.corpName as lotCorpName, lt.lotNumber as lotNumber, lt.registrationDate as registrationDate, prnt.corpName as parentCorpName, lt.project as project) "
+                + "lt.id as id, lt.corpName as lotCorpName, lt.lotNumber as lotNumber, lt.registrationDate as registrationDate, prnt.corpName as parentCorpName, lt.project as project, lt.chemist as chemist, vnd.name as vendorName, vnd.code as vendorCode, lt.supplier as supplier) "
                 + "FROM Lot AS lt "
                 + "JOIN lt.saltForm sltfrm "
                 + "JOIN sltfrm.parent prnt "
+                + "JOIN lt.vendor vnd "
```

Comment:
I think we want to do a `LEFT JOIN` here so we don't filter out lots with no vendor

Reply:
Good catch, did not think of that - thanks!

Reply:
Fixed it, tested it by removing vendor for a lot then hitting the API route, looks good!